### ISSUE TITLE
LLVM ERROR: Cannot select (with CMOV)

### ISSUE THREAD

**Issue by tkoenig1** [bagel99/llvm-my66000#11]
This time with the real my66000 compiler.  Error is:

```
LLVM ERROR: Cannot select: 0x5d50068: f32 = My66000ISD::CMOV 0x5d50958, 0x5d50478, 0x5d569b0
  0x5d50958: f32,ch = CopyFromReg 0x5d06608, Register:f32 %8
    0x5d50410: f32 = Register %8
  0x5d50478: f32,ch = CopyFromReg 0x5d06608, Register:f32 %9
    0x5d508f0: f32 = Register %9
  0x5d569b0: i64 = and 0x5d4fe60, Constant:i64<1>
    0x5d4fe60: i64,ch = CopyFromReg 0x5d06608, Register:i64 %7
      0x5d4fd90: i64 = Register %7
    0x5d562c8: i64 = Constant<1>
In function: cblas_isamax
PLEASE submit a bug report to https://bugs.llvm.org/ and include the crash backtrace.
Stack dump:
0.      Program arguments: llc --disable-lsr --enable-predication --enable-predication2 --enable-carry-generation --early-carry-coalesce --enable-vvm -march=my66000 isamax_opt.bc
1.      Running pass 'Function Pass Manager' on module 'isamax_opt.bc'.
2.      Running pass 'My66000 DAG->DAG Pattern Instruction Selection' on function '@cblas_isamax'
```
[tstcase.zip](https://github.com/bagel99/llvm-my66000/files/10117619/tstcase.zip)

clang is
```
tkoenig@gcc188:~/isamax> clang --version
clang version 12.0.0 (https://github.com/bagel99/llvm-my66000.git 9fb868de1043964594da55f145454316ba42a5ab)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /home/tkoenig/bin
```

**Comment by bagel99** [comment]
Fixed in commit d030246b63d36a6fb9cf3b6f5436e179ce074411.

**Comment by bagel99** [comment]
ixed in commit d030246b63d36a6fb9cf3b6f5436e179ce074411.
